test/inductor/test_torchinductor.py

```diff
@@ -1511,16 +1521,30 @@ def test_multilayer_var(self):
         def fn(a):
             return torch.var(a)
 
-        self.common(fn, ((torch.rand((10, 3, 352, 352), dtype=torch.float32),)))
-        self.common(fn, ((torch.rand((14923), dtype=torch.float32),)))
+        self.common(
+            fn,
+            ((torch.rand((10, 3, 352, 352), dtype=torch.float32),)),
+            rtol=1e-4,
+            atol=1e-4,
```

Comment:
These tolerances are quite high, can we limit them to only the relevant CPU cases.

Reply:
Set atol and rtol for the relevant CPU cases.